src/libshadertrap/src/executor.cc

```diff
@@ -33,6 +33,7 @@
 #include "libshadertrap/token.h"
 #include "libshadertrap/uniform_value.h"
 #include "libshadertrap/vertex_attribute_info.h"
+#include "libshadertrap/tokenizer.h"
```

Comment:
We definitely don't want the executor to depend on the tokenizer. The parser is the only class that should make use of the tokenizer - tokenizing is what happens at the beginning of processing a script. Execution only happens after a script has been fully parsed.

Reply:
@EGJ1996 I'm confused because you have resolved the comment, but you didn't enter any reply to it and the #include of tokenizer.h is still there.